packages/compass-web/src/entrypoint.tsx

```diff
@@ -62,6 +63,7 @@ import type {
   DebugFunction,
 } from './logger-and-telemetry';
 import { useCompassWebLoggerAndTelemetry } from './logger-and-telemetry';
+import { type TelemetryServiceOptions } from '@mongodb-js/compass-telemetry/dist/generic-track';
```

Comment:
Not sure if this type is exported, maybe we need to add the export to the file first, but we definitely shouldn't be importing from `dist` here

```suggestion
import { type TelemetryServiceOptions } from '@mongodb-js/compass-telemetry/provider';
```

Reply:
thanks for noticing!